test/cases/grid-template-areas.css

```diff
@@ -215,3 +215,27 @@
   /* autoprefixer: ignore next */
   grid-area: child;
 }
+
+.grid-example {
+  display: grid;
+}
+
+@media (max-width: 767.98px) {
+  .grid-example {
+    grid-template-areas:
+      'first first'
+      'second third';
+  }
+}
+
+.grid-area-first {
+  grid-area: first;
+}
+  
+.grid-area-second {
+  grid-area: second;
+}
+
+.grid-area-third {
+  grid-area: third;
+}
```

Comment:
Can you insert new line in both CSS files?

Reply:
Done 👍